lib/transaction/operation.go

```diff
@@ -77,7 +73,7 @@ func NewOperationFromInterface(oj OperationFromJSON) (op Operation, err error) {
 		if err != nil {
 			return
 		}
-		op.B = NewOperationBodyCreateAccount(body["target"].(string), amount)
+		op.B = NewOperationBodyCreateAccount(body["target"].(string), amount, body["linked"].(string))
```

Comment:
the operation is not revised yet. right?

Reply:
Revised in terms of what ?